### PR TITLE
Update test matrix

### PR DESCRIPTION
**Pull request by kevin-brown**
- [x] Remove Django 2.0
- [x] Remove Django 2.1
- [x] Add Django 3.2
- [x] Add Django 4.0
- [x] Add Python 3.9
- [x] Add Python 3.10

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -45,7 +46,7 @@ pytest-django = ">=4.1"
 pytest-cov = ">=2.10"
 pytest-isort = ">=1.2"
 pytest-flake8 = ">=1.0"
-pytest-black = ">=0.3"
+pytest-black = {version = ">=0.3", python = ">=3.6.2,<4"}
```
Comment (stephrdev) [inline pyproject.toml:49]: What is the reason for this change?

**Review by stephrdev — COMMENTED** [review]
Thank you for your contribution. Sadly, the 3.8 tests still fail and I have one question regarding the change of the pytest-black requirement.

**Comment by kevin-brown** [comment]
Yeah the Python 3.8 tests fail for some unknown reason related to the version pinning within Black itself (the tests pass when run independently). And since Black is unpinned, newer versions bumped up the minimum requirements for Python 3.6 to be 3.6.2 to mitigate some bugs. So that was carried down to pytest-black to not affect the overall package requirements.

File: pyproject.toml (hunk 1 above)
Comment (stephrdev) [reply pyproject.toml:49]: I think we can restrict the Python version for the whole cacheback project to at least 3.6.2. I had a similar issue in another project of mine: https://github.com/stephrdev/django-cruditor/blob/main/pyproject.toml#L26
With that, this change shouldn't be needed on the dependency entry.
